generators/app/index.js

```diff
@@ -0,0 +1,44 @@
+const Generator = require("yeoman-generator");
+module.exports = class extends Generator {
+   initializing() {
+      this.log("Creating RC Admin...");
+   }
+   async prompting() {
+
+      const {api_type} = await this.prompt([{
+         name : "api_type",
+         message : "What API dataProvider would you like to use? (REST or GraphQL)"
+      }]);
+      this.api_type = api_type;
+
+      const { registry } = await this.prompt([{
+         name: 'registry',
+         message: 'Which registry would you like to create?'
+      }]);
+      this.registry = registry;
+
+      const {entities} = await this.prompt([{
+         name : "entities",
```

Comment:
If Sundird RC is selected, cascaded questions are:
- Schema location
- APIs base URL